adapter/noop/noop.go

```diff
@@ -0,0 +1,87 @@
+// Copyright 2017 the Istio Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package noop
+
+import (
+	rpc "github.com/googleapis/googleapis/google/rpc"
+
+	"istio.io/mixer/adapter/noop/config"
+	"istio.io/mixer/pkg/adapter"
+)
+
+type (
+	builder struct{}
+	aspect  struct{}
+)
+
+// Register registers the no-op adapter as every aspect.
```

Comment:
So we can dynamically register no-op for all aspects with reflection, not sure if we really want to. It's not the worst IMO.